dashboard/test/models/lesson_test.rb

```diff
@@ -471,15 +471,10 @@ class LessonTest < ActiveSupport::TestCase
   end
 
   test 'summarize for script edit includes bonus levels' do
-    script = create :script, is_migrated: true
-    lesson_group = create :lesson_group, script: script
-    lesson = create :lesson, lesson_group: lesson_group, script: script, name: 'Lesson 1', key: 'lesson-1', relative_position: 1, absolute_position: 1
-    activity = create :lesson_activity, lesson: lesson
-    section = create :activity_section, lesson_activity: activity
-    level1 = create :level
-    level2 = create :level
-    create :script_level, script: script, lesson: lesson, activity_section: section, activity_section_position: 1, levels: [level1]
-    create :script_level, script: script, lesson: lesson, activity_section: section, activity_section_position: 2, levels: [level2], bonus: true
+    script = create :script, :with_levels
+    lesson = script.lessons.first
+    lesson.script_levels.last.update!(bonus: true)
+    lesson.reload
```

Comment:
this is another way to get a custom script level (or other object) -- use `with_levels` to create the entire tree, then use `update!` to update the part of the tree you want to change. remember to `.reload` after.